File: app/src/main/java/com/neome/java/api/core/deeplink/sig/SigDeeplinkHtml.java
// neome.ai API - do not change
//

package com.neome.java.api.core.deeplink.sig;

import com.neome.java.api.nucleus.base.sig.Sig;

import org.jetbrains.annotations.Nullable;

import java.util.Map;

@SuppressWarnings("unused")
public class SigDeeplinkHtml extends Sig
{
  @Nullable
  public String content;

  @Nullable
  public Map<String, String> contentHeaders;

  @Nullable
  public String contentType;

  @Nullable
  public Boolean isBase64Content;
}

File: app/src/main/java/com/neome/api/core/deeplink/sig/SigDeeplinkHtml.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.core.deeplink.sig

import java.util.Map
import com.neome.api.nucleus.base.sig.Sig

interface SigDeeplinkHtml : Sig
{
  val content: String?
  val contentHeaders: Map<String, String>?
  val contentType: String?
  val isBase64Content: Boolean?
}